www/js/services.js

```diff
@@ -281,10 +281,11 @@ angular.module('pvta.services', ['ngResource'])
   };
 })
 
-.service('Map', function($cordovaGeolocation){
+.factory('Map', function($cordovaGeolocation){
```

Comment:
I'm interested to know what you learned that caused you to change this.  It makes no difference to me because I can't determine a difference between `service` and `factory`, so I'm curious if you found something out.